src/stores/Entities/hooks.ts

```diff
@@ -1,131 +1,72 @@
+    }));
 };
 
 export const useEntitiesStore_capabilities_writable = () => {
-    return useZustandStore<EntitiesState, Schema>(
-        GlobalStoreNames.ENTITIES,
-        (state) => {
-            return {
-                ...state.capabilities.admin,
-                ...state.capabilities.write,
-            };
-        }
-    );
+    return useEntitiesStore((state) => {
+        return {
+            ...state.capabilities.admin,
+            ...state.capabilities.write,
+        };
+    });
 };
 
-export const useEntitiesStore_hydrateState = () => {
-    return useZustandStore<EntitiesState, EntitiesState['hydrateState']>(
-        GlobalStoreNames.ENTITIES,
-        (state) => state.hydrateState
-    );
-};
-export const useEntitiesStore_hydrated = () => {
-    return useZustandStore<EntitiesState, EntitiesState['hydrated']>(
-        GlobalStoreNames.ENTITIES,
-        (state) => state.hydrated
-    );
-};
-export const useEntitiesStore_setHydrated = () => {
-    return useZustandStore<EntitiesState, EntitiesState['setHydrated']>(
-        GlobalStoreNames.ENTITIES,
-        (state) => state.setHydrated
-    );
-};
-export const useEntitiesStore_setActive = () => {
-    return useZustandStore<EntitiesState, EntitiesState['setActive']>(
-        GlobalStoreNames.ENTITIES,
-        (state) => state.setActive
-    );
-};
-export const useEntitiesStore_hydrationErrors = () => {
-    return useZustandStore<EntitiesState, EntitiesState['hydrationErrors']>(
-        GlobalStoreNames.ENTITIES,
-        (state) => state.hydrationErrors
-    );
-};
-export const useEntitiesStore_setHydrationErrors = () => {
-    return useZustandStore<EntitiesState, EntitiesState['setHydrationErrors']>(
-        GlobalStoreNames.ENTITIES,
-        (state) => state.setHydrationErrors
-    );
-};
-export const useEntitiesStore_mutate = () => {
-    return useZustandStore<EntitiesState, EntitiesState['mutate']>(
-        GlobalStoreNames.ENTITIES,
-        (state) => state.mutate
-    );
+export const useEntitiesStore_capabilities_adminable = () => {
+    return useEntitiesStore((state) => state.capabilities.admin);
 };
-const useEntitiesStore_setMutate = () => {
-    return useZustandStore<EntitiesState, EntitiesState['setMutate']>(
-        GlobalStoreNames.ENTITIES,
-        (state) => state.setMutate
+
+export const useEntitiesStore_hasSupportRole = () => {
+    return useEntitiesStore(
+        useShallow((state) =>
+            Boolean(state.capabilities.admin[ESTUARY_SUPPORT_ROLE])
+        )
     );
 };
 
-export const useSidePanelDocsStore_resetState = () => {
-    return useZustandStore<EntitiesState, EntitiesState['resetState']>(
-        GlobalStoreNames.ENTITIES,
-        (state) => state.resetState
+export const useEntitiesStore_hasDemoTenantAccess = () => {
+    const readable = useEntitiesStore_capabilities_readable();
+    return useEntitiesStore(
+        useShallow(() => {
+            return Object.keys(readable).includes('demo/');
```

Comment:
The `demo/` tenant is referenced in a few places by name. Should it (and the `ops/` tenant) be stored in a constant as well?

Reply:
Yeah - I think we can do that in another PR. I feel like we should probably get some tenant helpers/hooks/whatever setup and store these off in const around that time.